charts/backingservices/README.md

```diff
@@ -31,18 +31,19 @@ If an externally managed elasticsearch cluster is being used, make sure the serv
 You may enable the component of [Elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch/values.yaml) in the backingservices by configuring the 'srs.srsStorage' section in values.yaml file to deploy ElasticSearch cluster automatically. For more configuration options available for each of the components, see their Helm Charts.
 
 Note: Pega does **not** actively update the elasticsearch dependency in `requirements.yaml`. To leverage SRS, you should:
```

Comment:
Note: Pega does **not** actively update the Elasticsearch dependency in `requirements.yaml`. To leverage SRS, you must do one of the following: